src/rprblender/properties/world.py

```diff
@@ -304,6 +304,17 @@ class RPR_EnvironmentProperties(RPR_Properties):
         size=3,
     )
 
+    # LIGHT GROUP AOV
+    group: EnumProperty(
+        name="Light Group",
+        items=(('1', "1", "Group 1"),
+               ('2', "2", "Group 2"),
+               ('3', "3", "Group 4"),
```

Comment:
“Group 3”

Reply:
It's ALMOST like I copy and pasted.  Fixed